.devcontainer/devcontainer.json

```diff
@@ -25,6 +25,7 @@
 	"forwardPorts": [
 		1927,
 		5432
+		// 8080
```

Comment:
Woops, just realized this was done deliberately here as part of this PR in the first place lol!

I think we should just clarify in the README specifically how to access the Swagger UI (both locally & via container) so it's clear for other devs who aren't familiar...

Reply:
Just mapped 8080 to 8080 and add a section in the readme to avoid confusion.